MC/bin/o2dpg_sim_workflow.py

```diff
@@ -800,17 +800,17 @@ def addQCPerTF(taskName, needs, readerCommand, configFilePath, objectsFile=''):
      ### MFT
      
      # to be enabled once MFT Digits should run 5 times with different configurations
-     #for flp in range(5):
-     #  addQCPerTF(taskName='mftDigitsQC' + str(flp),
-     #             needs=[det_to_digitask["MFT"]['name']],
-     #             readerCommand='o2-qc-mft-digits-root-file-reader --mft-digit-infile=mftdigits.root',
-     #             configFilePath='json://${O2DPG_ROOT}/MC/config/QC/json/qc-mft-digit-' + str(flp) + '.json',
-     #             objectsFile='mftDigitsQC.root')
+     for flp in range(5):
+       addQCPerTF(taskName='mftDigitsQC' + str(flp),
+                  needs=[det_to_digitask["MFT"]['name']],
+                  readerCommand='o2-qc-mft-digits-root-file-reader --mft-digit-infile=mftdigits.root',
+                  configFilePath='json://${O2DPG_ROOT}/MC/config/QC/json/qc-mft-digit-' + str(flp) + '.json',
+                  objectsFile='mftDigitsQC.root')
      
-     addQCPerTF(taskName='mftDigitsQC',
-                needs=[det_to_digitask["MFT"]['name']],
-                readerCommand='o2-qc-mft-digits-root-file-reader --mft-digit-infile=mftdigits.root',
-                configFilePath='json://${O2DPG_ROOT}/MC/config/QC/json/qc-mft-digit.json')
+     #addQCPerTF(taskName='mftDigitsQC',
+     #           needs=[det_to_digitask["MFT"]['name']],
+     #           readerCommand='o2-qc-mft-digits-root-file-reader --mft-digit-infile=mftdigits.root',
+     #           configFilePath='json://${O2DPG_ROOT}/MC/config/QC/json/qc-mft-digit.json')
```

Comment:
If you expect that this becomes obsolete now, please remove it instead of commenting out.